frontend/src/components/pages/unloaded/Unloaded.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import { CircularProgress } from '@material-ui/core';
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    loadContainer: {
+      display: "flex",
+      justifyContent: "center",
+    },
+  }),
+);
+
+/** Returns a spinner that indicates loading. */
```

Comment:
while this stresses me out as a mac user, this is an awesome new feature to address ambiguity while logging in! nice job!